.vscode/launch.json

```
@@ -7,39 +7,24 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "Graph",
-            "program": "${workspaceFolder}/lib/graph.js",
+            "name": "Visualize",
+            "program": "${workspaceFolder}/lib/main.js",
             "outFiles": [
 				"${workspaceFolder}/lib/**/*.js"
             ],
             "sourceMaps": true,
-            "args": [
-                "5"
-            ]
-        },
-        {
-            "type": "node",
-            "request": "launch",
-            "name": "Search",
-            "program": "${workspaceFolder}/lib/search.js",
-            "outFiles": [
-				"${workspaceFolder}/lib/**/*.js"
-            ],
-            "sourceMaps": true,
-            "args": [
-                "-id", "20", "16", "-inV", "15"
-            ]
+            "args": [ "visualize", ".\\example\\json.json", "--id", "2", "--inputFormat=json" ]
```

super nit: consider changing `json.json` to lsif-graph.json (PS I'm trying a new GitHub suggestion feature so apologies if the UI is weird for you 😃)
```suggestion
            "args": [ "visualize", ".\\example\\lsif-graph.json", "--id", "2", "--inputFormat=json" ]
```

---

I will give you my reasoning for naming it "json.json". The two example files are the same LSIF output. One is in the JSON format, the other on the JSON Lines format. The `--inputFormat` argument takes either "json" or "line", so that's what I named the files.

It sounds quite weird, so maybe we could go for "array.json"? I still want the name of the file to indicate the format we're using.

---

**Won't fix.** In the future, I want to add some unit tests regarding input format, and it will be clearer to have the files named after their formats. Yes, the "json.json" name is funky, but try to ignore the extension and focus on the name.